omero/developers/testing.txt

```diff
@@ -459,12 +472,6 @@ plugin :pypi:`pytest-pythonpath` can be used to add paths to
 
 .. program:: py.test
 
-.. option:: -s
-
-    This option allows the standard output to be shown on the console::
-
-        py.test test/integration/test_admin.py -s
```

Comment:
It is still offered by `py.test` but it was here as an example of something your couldn't do using `setup.py` The related PR now lets you use this option that way, which is easier, and so I thought the example no longer relevant and possibly confusing. We might, possibly, want to add another example(s) here.